src/scores.ts

```diff
@@ -62,127 +67,135 @@ export async function getScores(
   }
 }
 
+async function updateVotesVp(votes: any[], vpState: string, proposalId: string) {
+  const votesWithChange = votes.filter(vote => {
+    const key1 = sha256(JSON.stringify([vote.balance, vote.scores, vpState]));
+    const key2 = sha256(JSON.stringify([vote.vp, vote.vp_by_strategy, vote.vp_state]));
+    return key1 !== key2;
+  });
+  if (votesWithChange.length === 0) return;
+
+  const max = 200;
+  const pages = Math.ceil(votesWithChange.length / max);
+  const votesInPages: any = [];
+  Array.from(Array(pages)).forEach((x, i) => {
+    votesInPages.push(votesWithChange.slice(max * i, max * (i + 1)));
+  });
```

Comment:
Maybe better to use this, 
```suggestion
  for (let i = 0; i < pages; i++) {
    votesInPages.push(votesWithChange.slice(max * i, max * (i + 1)));
  }
```